### PR TITLE
Fix data not controlled by `var.enabled`

### LINKED ISSUE #9: Data not controlled by `enabled`
The following data seem not controlled by `var.enabled`

```
data "aws_iam_role" "masters" {
  name = "${var.masters_name}.${var.dns_zone}"
}

data "aws_iam_role" "nodes" {
  name = "${var.nodes_name}.${var.dns_zone}"
}
```

### PR DESCRIPTION
**Pull request by yujunz**
Closes #9

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -34,12 +34,12 @@ output "masters_security_group_id" {
 }
 
 output "masters_role_name" {
-  value       = "${data.aws_iam_role.masters.id}"
+  value       = "${data.aws_iam_role.masters.*.id}"
```
Comment (aknysh) [inline outputs.tf]:
```suggestion
  value       = "${join("", data.aws_iam_role.masters.*.id)}"
```

File: outputs.tf [hunk 2]
```diff
@@ -34,12 +34,12 @@ output "masters_security_group_id" {
...
   description = "K8s nodes Security Group ARN"
 }
 
 output "masters_role_arn" {
-  value       = "${data.aws_iam_role.masters.arn}"
+  value       = "${data.aws_iam_role.masters.*.arn}"
```
Comment (aknysh) [inline outputs.tf]:
```suggestion
  value       = "${join("", data.aws_iam_role.masters.*.arn)}"
```

File: outputs.tf [hunk 3]
```diff
@@ -54,11 +54,11 @@ output "nodes_security_group_id" {
 }
 
 output "nodes_role_name" {
-  value       = "${data.aws_iam_role.nodes.id}"
+  value       = "${data.aws_iam_role.nodes.*.id}"
```
Comment (aknysh) [inline outputs.tf]:
```suggestion
  value       = "${join("", data.aws_iam_role.nodes.*.id)}"
```

File: outputs.tf [hunk 4]
```diff
@@ -54,11 +54,11 @@ output "nodes_security_group_id" {
...
   description = "Kops nodes Role name"
 }
 
 output "nodes_role_arn" {
-  value       = "${data.aws_iam_role.nodes.arn}"
+  value       = "${data.aws_iam_role.nodes.*.arn}"
```
Comment (aknysh) [inline outputs.tf]:
```suggestion
  value       = "${join("", data.aws_iam_role.nodes.*.arn)}"
```

**Review by aknysh — CHANGES_REQUESTED** [review]
thanks @yujunz , looks good, please see a few comments.
Also, please run `terraform fmt` to format the code, and run

```
make init
make readme/deps
make readme
```

to rebuild README

thanks

**Comment by yujunz** [comment]
Done @aknysh 

There was an error when running `make readme`. I have to install gomplate manually to workaround it.

```
❯ make readme
curl --retry 3 --retry-delay 5 --fail -sSL -o /Users/yujunz/Code/github.com/cloudposse/terraform-aws-kops-metadata/build-harness/vendor/terraform-docs https://github.com/segmentio/terraform-docs/releases/download/v0.4.5/terraform-docs-v0.4.5-darwin-amd64 && chmod +x /Users/yujunz/Code/github.com/cloudposse/terraform-aws-kops-metadata/build-harness/vendor/terraform-docs
make: gomplate: No such file or directory
make: *** [readme/build] Error 1
```

**Comment by yujunz** [comment]
Is it possible to have GitHub Actions to take the chore of formatting and updating, e.g. suggest changes automatically after running `terraform fmt` and `make ...`.

**Review by aknysh — APPROVED** [review]
thanks @yujunz

**Pull request merged**